### PR TITLE
fix: unable to logout nextjs

### PR DESCRIPTION
**Pull request by kibo-geetanshuchhabra**

### CODE REVIEW

**Review by TriptiVermaXP — COMMENTED** [review]
Logout changes looks good, but found few issues on hamburger menu. While clicking on Log-in or Go-to-my-account, it's not closing the hamburger menu.
Translation is not happening  for order-status and wishlist. Also some hardcoded text present.

File: components/layout/AppHeader/KiboHeader.tsx [hunk 1]
```diff
@@ -346,8 +346,10 @@ export default function KiboHeader(props: KiboHeaderProps) {
     setAuthError('')
     if (!isAuthenticated) {
       showModal({ Component: LoginDialog })
+      handleHamburgerMenu(false)
     } else {
       router.push('/my-account')
+      handleHamburgerMenu(false)
```
Comment (TriptiVermaXP) [inline components/layout/AppHeader/KiboHeader.tsx:352]: move this line out of the if-else as same code written in both the blocks.

**Review by TriptiVermaXP — APPROVED** [review]
Looks good, please check one small comment.

File: components/layout/AppHeader/KiboHeader.tsx (hunk 1 above)
Comment (kibo-geetanshuchhabra) [reply components/layout/AppHeader/KiboHeader.tsx:352]: Resolved

**Review by kibo-sushant — APPROVED** [review]

**Review by kibo-kevinwatts — APPROVED** [review]

**Pull request merged**